gcloud-java-dns/src/test/java/com/google/gcloud/dns/it/ITDnsTest.java

```diff
@@ -952,4 +956,62 @@ public void testListDnsRecords() {
       clear();
     }
   }
+
+  @Test
+  public void testListZoneBatch() {
+    DNS.create(ZONE1);
+    DNS.create(ZONE_EMPTY_DESCRIPTION);
+    // the mock is used to check that the callbacks were not skipped
+    final RuntimeException callbackCalled = EasyMock.createStrictMock(RuntimeException.class);
+    EasyMock.expect(callbackCalled.initCause(null)).andReturn(callbackCalled);
+    EasyMock.expect(callbackCalled.initCause(null)).andReturn(callbackCalled);
+    EasyMock.expect(callbackCalled.initCause(null)).andReturn(callbackCalled);
+    EasyMock.replay(callbackCalled);
+    DnsBatch.Callback<Page<Zone>> callback = new DnsBatch.Callback<Page<Zone>>() {
```

Comment:
Why not mocking the callback and use `expect` and possibly [capture](http://easymock.org/api/org/easymock/EasyMock.html#capture-org.easymock.Capture-) to check that the calls are made correctly? You would no longer need the `callbackCalled` workaround.